src/p2p_plugin.c

```diff
@@ -40,30 +40,31 @@ extern int ncclIbRelaxedOrderingEnabled;
 NCCL_PARAM(SharpMaxComms, "SHARP_MAX_COMMS", 1);
 NCCL_PARAM(IbAdaptiveRouting, "IB_ADAPTIVE_ROUTING", -2);
 
-ncclResult_t pluginInit(ncclDebugLogger_t logFunction);
+ncclResult_t pluginInit_v7(ncclDebugLogger_t logFunction);
+ncclResult_t pluginInit_v6(ncclDebugLogger_t logFunction);
+ncclResult_t pluginInit_v5(ncclDebugLogger_t logFunction);
+
+ncclNet_v7_t ncclNetPlugin_v7 = {
+  "NCCL RDMA Plugin v7",
```

Comment:
Do you think this name should be more descriptive? I see this is following the old pattern

Reply:
this is a temporary name. It is overwritten when the actual plugin (ib/ucx) is assigned in line no 88.